.goreleaser.yml

```diff
@@ -67,6 +67,8 @@ scoops:
       name: scoop-bucket
       token: "{{ .Env.TAP_GITHUB_TOKEN }}"
     folder: bucket
+    shortcuts: [["openaq.exe", "openaq"]]
```

Comment:
seems like theres an extra set of square brackets here

Reply:
Thats how it works apparently
`shortcuts: [["drumroll.exe", "drumroll"]]`

Reply:
my bad i was looking at the depends array, double brackets makes sense here